### PR TITLE
draft: Update stats for contributor issue and prs

### PR DESCRIPTION
**Pull request by lwasser**
Ok this is a good start but there are several issues that i don't understand. 

1. The top bar links don't work as i expect them to. Docs [here](https://mystmd.org/guide/website-navigation) suggest that i can use a path to the file to link to a navigation file. I'm trying to create a drop-down top navigation for our metrics. Locally, however, when i build these links don't work.  
> DOCS SAY: A path to an internal page. This should begin with /.

```
site:
  nav:
    # top-level nav - these links don't work if you use the path to the file.
    - title: Peer review data
      url: /metrics/peer-review/current-reviews-data
      children:
        - title: Historical data
          url: /metrics/peer-review/historical-peer-review-stats
    - title: Contributor data
      url: /metrics/contributors/contributor-pr-issues
    - title: Contributing
      url: /README
```

Related to the above, I'm guessing Bootstrap doesn't default to the top-level nav url actually linking to anything. Is that true? if it is, then I think that should be in the docs, too. Regardless, because none of the top bar links are working for me now, I am not sure how navigation works. 

Screenshot 2024-08-27 at 6 10 34 PM

The sidebar works well, but I'm used to having a custom sidebar for every section or page on my site (think Sphinx). I don't understand how the sidebar complements the top bar nav or how to set things up so the tree for headings appears on the left-hand side. 

I'm looking at the docs [here](https://mystmd.org/guide/website-navigation#navigation-sidebar-secondary). If I use the top bar nav, I'd like to be able to hide the toc for the entire site but SHOW the toctree for the headings on the page on the left-hand side. I can't seem to get that to work. 

Screenshot 2024-08-27 at 6 11 20 PM

### CODE REVIEW

**Comment by agoose77** [comment]
> The sidebar works well, but I'm used to having a custom sidebar for every section or page on my site (think Sphinx). I don't understand how the sidebar complements the top bar nav or how to set things up so the tree for headings appears on the left-hand side.
> 
> I'm looking at the docs [here](https://mystmd.org/guide/website-navigation#navigation-sidebar-secondary). If I use the top bar nav, I'd like to be able to hide the toc for the entire site but SHOW the toctree for the headings on the page on the left-hand side. I can't seem to get that to work.

There are two sidebars, the _primary_ sidebar (left) which shows the table of contents, and the secondary sidebar (right) which shows the document outline. The TOC describes the pages and their relative hierarchy, whilst the document outline shows the headings. This is similar to pydata-sphinx-theme.

You do see the document outline when `hide_toc` is set to `true`, but presently it is hidden if you have only a single heading on the page. I think we should make that configurable, because it's a consistency feature.

File: myst.yml [hunk 1]
```diff
@@ -4,18 +4,30 @@ project:
...
+    # top-level nav - these links don't work if you use the path to the file.
+    - title: Peer review data
+      url: /metrics/peer-review/current-reviews-data
+      children:
+        - title: Historical data
+          url: /metrics/peer-review/historical-peer-review-stats
+        - title: Peer review data
+          url: /metrics/peer-review/current-reviews-data
+    - title: Contributor data
+      url: /metrics/contributors/contributor-pr-issues
```
[the remark this answers is not in the document]
Comment (lwasser) [reply myst.yml:25]: @agoose77 THANK YOU. i'll test this tonight. 

Does this call for a doc update?
[here](https://mystmd.org/guide/website-navigation) 

Do you agree, or am I misreading things?

Screenshot 2024-08-28 at 10 06 05 AM

 I read that example above as a full path to the file.

* follow-up question: if I wanted a slug to be custom like `metrics/contributor-pr-issues` here, would I add that to the front matter of the notebook?

**Comment by lwasser** [comment]
Thank you, @agoose77 so so much! I was fighting with this yesterday. 

I'll test this all out tonight. If you agree, I'm also happy to submit a small docs PR UNLESS the myst team would rather do something else.

**Comment by lwasser** [comment]
Ok, this is all working now. 

1. the toctree didn't appear on the right because, as you mentioned, I only had one h2. Once I added a second header, it worked! And the left side is now "hidden" infavor of the top bar drop downs. 
2. I think the docs need an update. Please let me know if I can help! Your review was incredibly helpful. Thank you again!!

**Comment by lwasser** [comment]
ok this is all working as expected now. I will merge!! 🚀  
let's see if it renders online as expected 🤞🏻 💗 !

**Comment by lwasser** [comment]
@all-contributors please add @agoose77 for code, review

**Pull request merged**